package/CHANGELOG

```diff
@@ -13,6 +13,25 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
+03/21/22 aya9aladdin
+
+* 2.2.0
+
+Fixes 
+   For the Cluster class: (Issue #3566)
```

Comment:
I'm not sure what @IAlibay or @tylerjereddy  thinks, but if you could make this a little more succinct that would be great.

While it is good to extensively document, we generally use a single point with a few lines per change. :)

Reply:
Agreed, test changes whilst great aren't user facing, in most cases we don't actually add changelog entries since they don't lead to changes that users need to know about.

I would either not include a changelog entry or make it very simple here, something like "Added tests for X and fixed minor error message typos" would be more than sufficient.

Reply:
I just removed this whole section from the CHANGELOG for simplicity